appveyor.yml

```diff
@@ -16,7 +16,7 @@ install:
       Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force | Out-Null
       Import-PackageProvider NuGet -Force | Out-Null
       Set-PSRepository -Name PSGallery -InstallationPolicy Trusted | Out-Null
-      Install-Module InvokeBuild -RequiredVersion 3.2.1 -Scope CurrentUser -Force | Out-Null
```

Comment:
So how did this break if we were specifying a **required** version of InvokeBuild?  Updates to InvokeBuild shouldn't have broken our build.

Reply:
It actually broke Travis who didn't have a required version.

Reply:
But that parameter is for `Install-Module`.  Oh, so maybe Travis is using a version of PowerShellGet that doesn't have that parameter?  Er, that doesn't make sense because this next line uses this parameter:
```
Install-Module platyPS -RequiredVersion 0.7.6 -Scope CurrentUser -Force | Out-Null 
```
I think it is OK to update occassionally to the latest version of InvokeBuild.  I just don't know that we want to do that everytime we build. In fact, the `-RequiredVersion` parameter is meant to prevent the very error we encountered - "new release of InvokeBuild breaks our build".

OK, I think I see what is going on here.  It is the travis build that is breaking, right?  That build doesn't use `appveyor.yml`.  It uses `.travis.yml` which runs `build\travis.ps1`.  And that script has this line in it:
``
Install-Module InvokeBuild -Scope CurrentUser -Force
```
It seems to me that we need to add `-RequiredVersion` to that file and bump it now to `5.0.0` and put the parameter back in appveyor.yml but bump it up to `5.0.0` as well.

Reply:
Instead of `-RequiredVersion` I've specified `-MaximumVersion` so that we can still get patch updates without the harsh breaking changes of a top level version number (v5 -> v6 for example).

Let me know if you feel strongly otherwise 😄

Reply:
At least for `platyPS` we should stick with `-RequiredVersion` because they haven't hit 1.0 yet.  I don't recall them having any big breaking changes, but there isn't anything stating they won't afaict

Reply:
@SeeminglyScience I've updated it everywhere to require the `0.9.0` version of the package. This is the version that Travis was using. For Windows, this means it has been updated from `0.7.6` to `0.9.0`. Yay consistency!